packages/relay/src/lib/eth.ts

```diff
@@ -902,6 +902,16 @@ export class EthImpl implements Eth {
 
         this.hapiService.decrementErrorCounter(e.statusCode);
         this.logger.error(e, `${requestIdPrefix} Error raised during getCode for address ${address}, err code: ${e.statusCode}`);
+      } if(e instanceof PrecheckStatusError) {
```

Comment:
nit: should there be a new line here?
Is this at the same level of the SDKCLientError check?

Reply:
Or is this an `else if`?

Reply:
hmmm, yeah, it would be either SDKCLientError or PrecheckStatusError... so even if it works `as is` the `else if` would be better.